server/src/test/java/com/broadcom/lsp/cobol/usecases/TestExtraneousInputEOFExpecting.java

```diff
@@ -35,11 +35,10 @@
           + "           if (1 > 0) NEXT SENTENCE{|1}"; // No dot at the end of file
 
   private static final String MESSAGE =
-      "Extraneous input '<EOF>' expected {ACCEPT, CANCEL, CLOSE, COMPUTE, DISPLAY, DIVIDE, EVALUATE, EXHIBIT, "
-          + "GOBACK, IF, INITIALIZE, INITIATE, INSPECT, MULTIPLY, PERFORM, PURGE, SERVICE, SORT, STOP, STRING, "
-          + "SUBTRACT, TERMINATE, UNSTRING, CALL, CONTINUE, DISABLE, ENABLE, EXIT, GENERATE, GO, MERGE, SEARCH, "
-          + "XML, ADD, ALTER, DELETE, ENTRY, MOVE, OPEN, READ, RECEIVE, RELEASE, RETURN, REWRITE, SEND, SET, START, "
-          + "SQL, EXEC_SQL, EXEC, WRITE, DOT_FS}";
+      "Extraneous input '<EOF>' expected {ACCEPT, CANCEL, CLOSE, COMPUTE, DISPLAY, DIVIDE, EVALUATE, EXHIBIT, GOBACK,"
```

Comment:
I would recommend providing a special message for EOF, something like "Unexpected end of file". Up to @zimlu02

Reply:
I've never liked this long error msg. So the option would be "Unexpected end of file" with no suggestion? I would agree with it.